src/lib/components/forms/input/color-picker/color.helper.ts

```diff
@@ -230,57 +209,46 @@ export class Color
      * Read a color formatted as HSL.
      * @param color ColorHSL The color formatted as HSL
      */
-    private readHSL(color:ColorHSL):void
-    {
-        let r:number = color.l;
-        let g:number = color.l;
-        let b:number = color.l;
-
-        if(color.s !== 0)
-        {
-            let q:number;
-            if(color.l < 0.5)
-            {
+    private readHSL(color: ColorHSL): void {
+        let r: number = color.l;
+        let g: number = color.l;
+        let b: number = color.l;
+
+        if (color.s !== 0) {
+            let q: number;
+            if (color.l < 0.5) {
                 q = color.l * (1 + color.s);
-            }
-            else
-            {
+            } else {
                 q = color.l + color.s - color.l * color.s;
             }
 
-            let p:number = 2 * color.l - q;
+            let p: number = 2 * color.l - q;
 
-            r = this.hue2rgb(p, q, color.h + (1 / 3));
+            r = this.hue2rgb(p, q, color.h + 1 / 3);
             g = this.hue2rgb(p, q, color.h);
-            b = this.hue2rgb(p, q, color.h - (1 / 3));
+            b = this.hue2rgb(p, q, color.h - 1 / 3);
         }
 
         this.r = Math.round(r * 255);
         this.g = Math.round(g * 255);
         this.b = Math.round(b * 255);
     }
 
-    private hue2rgb(p:number, q:number, t:number):number
-    {
-        if(t < 0)
-        {
+    private hue2rgb(p: number, q: number, t: number): number {
+        if (t < 0) {
             t += 1;
         }
-        if(t > 1)
-        {
+        if (t > 1) {
             t -= 1;
         }
-        if(t < (1 / 6))
-        {
+        if (t < 1 / 6) {
```

Comment:
vorher war es:  **if(t < (1 / 6))**  .. stimmt die Rechnung jetzt noch so? 😅

Reply:
Ich gucke mal, ob ich was rausfinden, in welcher Reihenfolge die Operatoren ausgeführt werden 🤔

Reply:
Laut der Tabelle hier https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Operators/Operator_Precedence#Tabelle
Hat die Division eine Prio von 14 und Kleiner eine von 11. Also wird weiterhin die Division zuerst ausgeführt 🙂

Reply:
Ok danke 😄  genau das war meine bedenken, aber ansonsten top 💪